{
  "1.0.4": [
    "<ru.fix.armeria.commons.RetryRulesKt: void <clinit>()>",
    "<ru.fix.armeria.commons.ArmeriaExceptionsHelperKt: java.lang.Throwable getActualResponseExceptionCause(java.lang.Throwable)>"
  ],
  "1.0.3": [
    "<ru.fix.armeria.commons.RetryRulesKt: void <clinit>()>",
    "<ru.fix.armeria.commons.ArmeriaExceptionsHelperKt: java.lang.Throwable getActualResponseExceptionCause(java.lang.Throwable)>"
  ],
  "1.1.1": [
    "<ru.fix.armeria.commons.RetryRulesKt: void <clinit>()>",
    "<ru.fix.armeria.commons.ArmeriaExceptionsHelperKt: java.lang.Throwable getActualResponseExceptionCause(java.lang.Throwable)>"
  ],
  "1.1.0": [
    "<ru.fix.armeria.commons.RetryRulesKt: void <clinit>()>",
    "<ru.fix.armeria.commons.ArmeriaExceptionsHelperKt: java.lang.Throwable getActualResponseExceptionCause(java.lang.Throwable)>"
  ],
  "1.0.2": [
    "<ru.fix.armeria.commons.RetryRulesKt: void <clinit>()>",
    "<ru.fix.armeria.commons.ArmeriaExceptionsHelperKt: java.lang.Throwable getActualResponseExceptionCause(java.lang.Throwable)>"
  ],
  "1.0.1": [
    "<ru.fix.armeria.commons.RetryRulesKt: void <clinit>()>",
    "<ru.fix.armeria.commons.ArmeriaExceptionsHelperKt: java.lang.Throwable getActualResponseExceptionCause(java.lang.Throwable)>"
  ]
}